components/DisplayItemDetailsClient.tsx

```diff
@@ -0,0 +1,101 @@
+'use client';
+
+import Image from 'next/image';
+import ItemDetails from '@/components/ItemDetails';
+import PostageOptionDisplay from '@/components/PostageOptionDisplay';
+import NewConversationButton from '@/components/buttons/NewConversationButton';
+
+import BackButton from '@/components/buttons/BackButton';
+import ButtonRounded from './buttons/ButtonRounded';
+import { updateRequestToReserve } from '@/supabase/models/updateRequestToReserve';
+import { PartialItem } from '@/types/supabaseTypes';
+import { useState } from 'react';
+
+type ItemDetailsClientProps = {
+  item: PartialItem;
+  user: { id: string };
+  canMessage: boolean;
+  donorId: string;
+  donorEmail: string;
+  donorName: string;
+  title: string;
+};
+
+const DisplayItemDetailsClient: React.FC<ItemDetailsClientProps> = ({
+  item,
+  user,
+  canMessage,
+  donorId,
+  donorEmail,
+  donorName,
+  title,
+}) => {
+  const [requestStatus, setRequestStatus] = useState<string | null>(null);
```

Comment:
I find the evaluation `requestStatus && {...}` confusing because "status" seems to imply one of many possible values but this usage makes it seem a boolean. Can we just call it `[message, setMessage]` ?

Reply:
Done